--- ctk-0.1.0+git20131105~b8307961b6.orig/CMakeLists.txt
+++ ctk-0.1.0+git20131105~b8307961b6/CMakeLists.txt
@@ -369,6 +369,8 @@ if(CMAKE_COMPILER_IS_GNUCXX)
   endif()
 
   set(CTK_C_FLAGS "${cflags} ${CTK_C_FLAGS}")
+  # Fix for debian bug #637687, should never be merged in CTK master
+  set(CTK_CXX_FLAGS "${CTK_CXX_FLAGS} -DHAVE_CONFIG_H")
   set(CTK_CXX_FLAGS "${cflags} -Woverloaded-virtual -Wold-style-cast -Wstrict-null-sentinel -Wsign-promo ${CTK_CXX_FLAGS}")
 endif()
 
